xml/ha_migration.xml

```diff
@@ -708,6 +724,111 @@
    </important>
    <para>The &hawk2; <guimenu>Status</guimenu> screen also shows a warning if
     different CRM versions are detected for your cluster nodes.</para>
+   <para>Beside an in-place upgrade, many customers prefer a fresh installation
+    even for moving to the next service pack. The following procedure shows a
+    scenario where a two-node cluster with the nodes &node1; and &node2; is
+    upgraded to the next service pack (SP):
+   </para>
+   <procedure xml:id="pro-ha-migration-rolling-upgrade-newsp-freshinstall">
+    <title>Performing a Cluster-wide Fresh Installation of a New Service Pack</title>
+    <step xml:id="st-ha-migration-rolling-up-sp-backup">
+     <para>Make a backup of your cluster configuration. A minimum set of files
+      are shown in the following list<!--<xref linkend="ex.ha.migration.minlist4backup"
+       xrefstyle="select:label"/>-->:
+     </para>
+     <!--<example xml:id="ex.ha.migration.minlist4backup">
+      <title>Minimum Files for Backup</title>-->
+     <screen>/etc/corosync/corosync.conf
+/etc/corosync/authkey
+/etc/sysconfig/sbd
+/etc/modules-load.d/watchdog.conf
+/etc/hosts
+/etc/ntp.conf</screen>
+     <!--</example>-->
+     <para>Depending on your resources, you may also need the following files:</para>
+     <!--<example xml:id="ex.ha.migration.extlist4backup">
+      <title>Extended List of Files for Backup</title>-->
+     <screen>/etc/services
+/etc/passwd
+/etc/shadow
+/etc/groups
+/etc/drbd/*
+/etc/lvm/lvm.conf
+/etc/mdadm.conf
+/etc/mdadm.SID.conf</screen>
+     <!--</example>-->
+    </step>
+    <step xml:id="st-ha-migration-rolling-alice">
+     <para>Start with node &node1;.
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Make sure no resources are running. If needed,
+        migrate your resource to &node2;. See <xref
+         linkend="sec-conf-hawk2-manage-migrate" xrefstyle="select:label"/>
+        (&hawk;) or <xref
+         linkend="sec-ha-manual-config-migrate" xrefstyle="select:label"/>
+        (&crmshell;)  for details.
+       </para>
+      </step>
+      <step>
+       <para>
+        Check that the cluster is healthy and idle. All resources must be fine.
+       </para>
+      </step>
+      <step>
+       <para>
+        Set both nodes in maintenance mode (see
+        <xref linkend="sec-ha-maint-mode-cluster"/>).
```

Comment:
By putting cluster/nodes into maintenance mode,  we could stop cluster service while the resources remain running without being touched by cluster.  I don't think we specifically need this. Since by doing fresh installation, we have to reboot the systems anyway...

Reply:
> I don't think we specifically need this in here.

Ok, removed this step.